Sources/StreamChatUI/Appearance+Images.swift

```diff
@@ -44,6 +44,7 @@ public extension Appearance {
         public var newChannel: UIImage = UIImage(named: "pencil", in: .streamChatUI)!
         public var folder: UIImage = UIImage(named: "folder", in: .streamChatUI)!
         public var restart: UIImage = UIImage(named: "restart", in: .streamChatUI)!
+        public var download: UIImage = UIImage(named: "download", in: .streamChatUI)!
```

Comment:
Can we use the default system icon for iOS13+ ?

Reply:
Yes indeed! We can use `icloud.and.arrow.down`, done ✅